test/memory/lifetime/allocation.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: Unmanaged-Time:, Shared-Time:, Owned-Time:
+files: allocation.dat, allocation.dat, allocation.dat
+graphkeys: Unmanaged, Shared, Owned
+ylabel: Time (milliseconds)
+graphtitle: Lifetime Management Overhead (Unmanaged as Baseline)
```

Comment:
Maybe something like "Serial allocation styles"  or "Serial empty-object allocation styles"? (So we can have a vague sense of what it does without opening the test)